### ISSUE TITLE
Manual Installation problems + conda

### ISSUE THREAD

**Issue by Kamranahm** [Gourieff/comfyui-reactor-node#70]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/comfyui-reactor-node/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

Running a manual install of Comfyui in a conda environment. Everything is working fine, but can't install this add-on. I dive into the reactor directory and run "python install.py" But get an error...

Hoping for a fix or some guidance on how to get around this. 

### Steps to reproduce the problem

Your workflow

created conda environment running python 3.11
cloned comfyui
installed pytorch/torchvision/torchaudio with cuda 12.1
installed all requirements for comfyui manually
cloned reactor in custom_nodes folder
changed directory to reaktor folder and ran "python install.py"
get error

### Sysinfo

Windows 10, comfyui is working flawlessly with multiple custom nodes. So isolated issue.

### Relevant console log

```Shell
(D:\ai\comfy) D:\ai\comfy\ComfyUI\custom_nodes\comfyui-reactor-node>python install.py
D:\ai\comfy\ComfyUI\custom_nodes\comfyui-reactor-node\install.py:3: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
Traceback (most recent call last):
  File "D:\ai\comfy\ComfyUI\custom_nodes\comfyui-reactor-node\install.py", line 7, in <module>
    import folder_paths
ModuleNotFoundError: No module named 'folder_paths'
```

**Comment by Gourieff** [comment]
At first you need to activate your Conda, e.g.:
`.\comfy\conda\Scripts\activate` or from any other folder you use
Then install requirements manually:
`pip install -r D:\ai\comfy\ComfyUI\custom_nodes\comfyui-reactor-node\requirements.txt`

**Comment by ffdown** [comment]
> At first you need to activate your Conda, e.g.: `.\comfy\conda\Scripts\activate` or from any other folder you use Then install requirements manually: `pip install -r D:\ai\comfy\ComfyUI\custom_nodes\comfyui-reactor-node\requirements.txt`

no way... after this action... 

`*** Error loading script: reactor_swapper.py
    Traceback (most recent call last):
      File "E:\AI\StableDiff\modules\scripts.py", line 382, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
      File "E:\AI\StableDiff\modules\script_loading.py", line 10, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "E:\AI\StableDiff\extensions\comfyui-reactor-node\scripts\reactor_swapper.py", line 15, in <module>
        import folder_paths
    ModuleNotFoundError: No module named 'folder_paths'`

**Comment by Kamranahm** [comment]
FFDown

> > At first you need to activate your Conda, e.g.: `.\comfy\conda\Scripts\activate` or from any other folder you use Then install requirements manually: `pip install -r D:\ai\comfy\ComfyUI\custom_nodes\comfyui-reactor-node\requirements.txt`
> 
> no way... after this action... and my system not comfyui it's Stable Diff
> 

Not sure if you are from a different issue post or something. But this issue I opened was specifically related to comfyui, you may be posting in the wrong thread.

Anyways, back on point.

I ended up running the roop comfyui addon and it actually installed the requirements which line up mostly with yours...Even though I already manually installed them via conda packages. I did as you asked as well and it already had all the requirements installed in my activated conda environment. So that part was taken care of....But still got a swap_model not found error in your node.

I opened up your install.py file in notepad++ and noticed the models/insight directory being noted, and put one and two together and put the inswapper model in a created folder there and got it to work. There is definitely an issue with the install script though. Not sure what it is. I don't think that the mobile version of comfyui would be any different in running the script, but I may be mistaken there. But I dislike running anything not in a conda environment to keep them isolated, so here I dug the hole =P. 

I powered up my spare machine and tried another conda environment there with your install.py file...still failed, same error. Even running the current script now even though your node in working in comfyui still shoots the same error I posted previously. If I had to guess, this is all speculation of course....It's failing to load the folder_paths.py file in the comfyui directory? Just guessing here...I'm not a coder, just putting one and two together. 

But yea...If I wasn't tech savvy and able to dissect the problem, would of never got this working, as there is no manual instructions incase problems arise on what to put where. I doubt most beginners would go the conda route, so probably an isolated situation, but might be worthwhile to put manual instructions as well with the directory listed for needed files to be safe.

**Comment by abishek-baskaran** [comment]
Had the same issue. I just copied folder_paths.py from ComfyUI folder into custom_nodes/comfyui-reactor-node and then it worked.

**Comment by Kamranahm** [comment]
> Had the same issue. I just copied folder_paths.py from ComfyUI folder into custom_nodes/comfyui-reactor-node and then it worked.

Much appreciated. Good to know. I'll mark as closed considering it's been brought to light. Thanks.

**Comment by Gourieff** [comment]
> I ended up running the roop comfyui addon and it actually installed the requirements which line up mostly with yours...Even though I already manually installed them via conda packages. I did as you asked as well and it already had all the requirements installed in my activated conda environment. So that part was taken care of....But still got a swap_model not found error in your node.

Yeah, I forgot to spot that thing about downloading the inswapper model if you install manually

> I opened up your install.py file in notepad++ and noticed the models/insight directory being noted, and put one and two together and put the inswapper model in a created folder there and got it to work. There is definitely an issue with the install script though. Not sure what it is. I don't think that the mobile version of comfyui would be any different in running the script, but I may be mistaken there. But I dislike running anything not in a conda environment to keep them isolated, so here I dug the hole =P.

It worked with no errors before, I will tested it again, there could be the issue because of the last update

**Comment by Gourieff** [comment]
@Kamranahm 
Fixed https://github.com/Gourieff/comfyui-reactor-node/commit/a65039bd830fd22099b029e6574970f62ebdbfc1
I will appreciate if you try it again

**Comment by Gourieff** [comment]
> no way... after this action... and my system not comfyui it's Stable Diff
> 
> `*** Error loading script: reactor_swapper.py Traceback (most recent call last): File "E:\AI\StableDiff\modules\scripts.py", line 382, in load_scripts script_module = script_loading.load_module(scriptfile.path) File "E:\AI\StableDiff\modules\script_loading.py", line 10, in load_module module_spec.loader.exec_module(module) File "<frozen importlib._bootstrap_external>", line 883, in exec_module File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed File "E:\AI\StableDiff\extensions\comfyui-reactor-node\scripts\reactor_swapper.py", line 15, in <module> import folder_paths ModuleNotFoundError: No module named 'folder_paths'`

According to the log, you're try to install this node directly into the SD WebUI, it doesn't work like this
You have to use ComfyUI as standalone app or as the sd-webui extension - and install the node inside ComfyUI
If you need the extension for SD WebUI - here it is https://github.com/Gourieff/sd-webui-reactor

**Comment by SuperUserNameMan** [comment]
> @Kamranahm Fixed [a65039b](https://github.com/Gourieff/comfyui-reactor-node/commit/a65039bd830fd22099b029e6574970f62ebdbfc1) I will appreciate if you try it again

I think there is something bizarre happening with this patch during fresh install into confyui. 

I found `./insightface/inswapper_128.onnx` was created into `./custom_nodes/models/` instead of into `./models/`.

If I leave it that way, the node remains red, and I can't select the `swap_model`.

If I move `./insighface/` into `./models/` then i can select `swap_model`, the node works, and it downloads `./models/insightface/models/buffalo_l/*`.

**Comment by Gourieff** [comment]
> > @Kamranahm Fixed [a65039b](https://github.com/Gourieff/comfyui-reactor-node/commit/a65039bd830fd22099b029e6574970f62ebdbfc1) I will appreciate if you try it again
> 
> I think there is something bizarre happening with this patch during fresh install into confyui.
> 
> I found `./insightface/inswapper_128.onnx` was created into `./custom_nodes/models/` instead of into `./models/`.
> 
> If I leave it that way, the node remains red, and I can't select the `swap_model`.
> 
> If I move `./insighface/` into `./models/` then i can select `swap_model`, the node works, and it downloads `./models/insightface/models/buffalo_l/*`.

Geez, I shouldn't have done that at night
Fixed 🤞 https://github.com/Gourieff/comfyui-reactor-node/commit/7c262d635a1306e957029c46a9d62cf898ac12a8
